### PR TITLE
Definfo ui update v2

### PR DESCRIPTION
**Pull request by ggilmore**
DefInfo Look and Feel Updates

### CODE REVIEW

File: app/web_modules/sourcegraph/repo/styles/Repo.css [hunk 1]
```diff
@@ -47,3 +47,9 @@
 	composes: col-9 from grid;
 	composes: modal from "sourcegraph/components/styles/modal.css";
 }
+
```
Comment (renfredxh) [inline app/web_modules/sourcegraph/repo/styles/Repo.css]: Nitpicky style comment: we usually add a space before the `{` for styles or blocks, e.g. `.outer-container {`

File: app/web_modules/sourcegraph/def/RepoRefsContainer.js [hunk 2]
```diff
@@ -40,12 +41,16 @@ class RepoRefsContainer extends Container {
...
 		state.defRepos = props.defRepos || [];
+		state.repoCallback = props.repoCallback || null;
 		state.refLocations = state.def ? DefStore.getRefLocations({
 			repo: state.repo, commitID: state.commitID, def: state.def, repos: state.defRepos,
 		}) : null;
 		if (state.refLocations && state.refLocations.PagesFetched >= state.currPage) {
 			state.nextPageLoading = false;
 		}
+		if (state.repoCallback && state.defRepos) {
+			// state.repoCallback(state.defRepos);
```
Comment (renfredxh) [inline app/web_modules/sourcegraph/def/RepoRefsContainer.js]: Remove the `repoCallback` stuff if it's not needed anymore

File: app/web_modules/sourcegraph/repo/styles/Repo.css (hunk 1 above)
Comment (ggilmore) [inline app/web_modules/sourcegraph/repo/styles/Repo.css]: Fixed.

**Comment by renfredxh** [comment]
LGTM

**Pull request merged**